app/lib/efile/nj/nj1040_calculator.rb

```diff
@@ -42,7 +42,11 @@ def calculate
         set_line(:NJ1040_LINE_41, :calculate_line_41)
         set_line(:NJ1040_LINE_42, :calculate_line_42)
         set_line(:NJ1040_LINE_43, :calculate_line_43)
+        set_line(:NJ1040_LINE_45, :calculate_line_45)
+        set_line(:NJ1040_LINE_49, :calculate_line_49)
```

Comment:
Going to not change this because there will eventually be calculation logic, it won't always be 0, so would prefer to follow existing function pattern